### PR TITLE
[AP-3076] seed service levels

### PR DESCRIPTION
**Pull request by jsugarman**
## What
Seed service levels and proceeding type service levels tables

[Link to story](https://dsdmoj.atlassian.net/browse/AP-3076)

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: spec/models/proceeding_type_spec.rb [hunk 1]
```diff
@@ -16,9 +19,18 @@
...
       expect(service_levels).to match_array [service_level1, service_level2]
     end
+
+    it "includes join table boolean attribute `proceeding_default`" do
+      expect(service_levels.first).to have_attributes(proceeding_default: be_in([true, false]))
+    end
+
+    it "responds to proceeding_default" do
+      expect(service_levels.first).to respond_to(:proceeding_default)
+      expect(service_levels.first.proceeding_default).to be_in([true, false])
```
Comment (jsugarman) [inline spec/models/proceeding_type_spec.rb:32]: use an `all` qualifier instead

File: db/seed_data/proceeding_type_service_levels.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+# List of proceedig type service levels join table records
```
Comment (colinbruce) [inline db/seed_data/proceeding_type_service_levels.yml:2]:
```suggestion
# List of proceeding type service levels join table records
```

File: db/seed_data/proceeding_type_service_levels.yml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+# #uuid2 1, ”Family Help (Higher)”, 1
+#
+# TABLE proceeding_type_service_levels
+# id, service_level_id, proceeding_type_id, default
+# [#, #uuid1, each domestic abuse proceeding id, true] * all DA proceedings
+# [#, #uuid1, each section 8 proceeding id, false] * all section 8 proceedings
+# [#, #uuid2, each section 8 proceeding id, true] * all section 8 proceedings
+#
+
+# Full Representation, domestice proceeding, default
```
Comment (colinbruce) [inline db/seed_data/proceeding_type_service_levels.yml:19]:
```suggestion
# Full Representation, domestic abuse proceeding, default
```

File: spec/rails_helper.rb [hunk 4]
```diff
@@ -69,21 +77,24 @@
...
+    end
+  end
 end
 
 def seed_live_data
-  Dir[Rails.root.join("db/populators/*.rb")].sort.each do |seed_file|
-    require seed_file
+  # you get lots of <class> or <constant> already initialized/defined warnings without this silencer
+  Kernel.silence_warnings do
+    load Rails.root.join("db/seeds.rb")
```
Comment (colinbruce) [inline spec/rails_helper.rb:96]: Can this guarantee sequence? 
I know we have had issues in the past with can't seed X because Y isn't populated yet

**Review by colinbruce — COMMENTED** [review]
Some minor typos and an observation on the object load refactor

File: spec/rails_helper.rb (hunk 4 above)
Comment (jsugarman) [reply spec/rails_helper.rb:96]: not sure, although the sequence here will be exactly what the running app would on pod spin up (`docker_scripts/setup_db`, do so I'd hope so 😬 . To me this has the added benefit of breaking tests if the order is wrong for the app seeding.

Comment (colinbruce) [reply spec/rails_helper.rb:96]: Discussed and happy to merge as is and monitor :+1:

**Review by colinbruce — APPROVED** [review]
:+1:

**Pull request merged**